index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="about.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
+      integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <script src="html.js" defer></script>
+    <script src="speakers.js" defer></script>
+    <script src="showmore.js" defer></script>
+    <title>Capstone by Enoh</title>
+  </head>
+  <body>
+    <div class="body-container">
+      <header class="header">
+        <div class="banner">
+          <ul class="banner-list">
+            <li>
+              <a href="#"><i class="fa-brands fa-facebook"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My page</a></li>
+            <li><a href="#">logout</a></li>
+          </ul>
+        </div>
+        <nav class="navbar">
+          <div class="navbar-container">
+            <a href="./index.html"
+              ><img src="./images/UN_emblem_blue.png" alt="" class="logo"
+            /></a>
+
+            <div class="hamburger"><i class="fa-solid fa-bars"></i></div>
+            <ul class="menu-list">
+              <li><a href="./about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#">Join</a></li>
+              <li class="cc-campaign"><a href="#">CC Campaign</a></li>
+            </ul>
+            <ul class="mobile-menu">
+              <div class="closebutton"><i class="fa-solid fa-x"></i></div>
+              <li><a href="./index.html">Home</a></li>
+              <li><a href="./about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#">Join</a></li>
```

Comment:
- [OPTIONAL] I put all my comments as optional because the first reviewer doesn’t mention them but they're very important. I notice that you don't use the given colors for the website "#272a31, #ec5242, #d3d3d3" and the font family should be `COCOGOOSE and LATO`.

<img width="338" alt="Screenshot 2023-08-09 at 15 34 35" src="https://github.com/enocol/capstone-project/assets/74501523/248ecb16-0bbc-44bd-a83d-aec611dd1db4">

<img width="647" alt="Screenshot 2023-08-09 at 15 39 55" src="https://github.com/enocol/capstone-project/assets/74501523/f72c315b-8ce5-4684-a1dc-ed4131626cea">

- [OPTIONAL] Your link color is different from the original design, please kindly fix that 🙃 

Your design

<img width="1025" alt="Screenshot 2023-08-09 at 15 21 34" src="https://github.com/enocol/capstone-project/assets/74501523/56886cb5-9058-4844-b0ad-e8982a91ee17">

Original design

<img width="981" alt="Screenshot 2023-08-09 at 15 22 33" src="https://github.com/enocol/capstone-project/assets/74501523/0c9b696f-b5b6-4026-80bd-35a9d077346d">

Reply:
@cynthiainga Thank you for your comments. I have taken your comment into account and will definitely be seeing into the changes. They are an easy fix, so thank you very much. Cheers.